modules/rosa-cluster-autoscaler-cli-settings.adoc

```diff
@@ -0,0 +1,114 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa-cluster-autoscaling.adoc
+// * osd_cluster_admin/osd-cluster-autoscaling.adoc
+
+:_content-type: REFERENCE
+[id="rosa-cluster-cli-autoscale-settings_{context}"]
+= Cluster autoscaling settings using the ROSA CLI
+
+There are several options you can add to the cluster creation command to adjust autoscaling options. The tables explain all the configurable autoscaler settings when using the ROSA CLI (`rosa`).
+
+[cols="4",options="header"]
+|===
+|Setting
+|Description
+|Type or Range
+|Example/Description
+
+|`--autoscaler-balance-similar-node-groups`
+|Identify node groups with the same instance type and label set, and aim to balance respective sizes of those node groups.
+|`boolean`
+|Add it to set to true, do not add to set to false.
+
+|`--autoscaler-skip-nodes-with-local-storage`
+|If true, cluster autoscaler will never delete nodes with pods with local storage, for example, EmptyDir or HostPat.
+|`boolean`
+|Add it to set to true, do not add to set to false.
+
+|`--autoscaler-log-verbosity _int_`
+|Autoscaler log level. Replace _int_ in the command with the number you want to use.
+|`integer`
+|`--autoscaler-log-verbosity 4`
+
+|`--autoscaler-max-pod-grace-period _int_`
+|Gives pods graceful termination time before scaling down, measured in seconds. Replace _int_ in the command with the number you want to use.
+|`integer`
+|`--autoscaler-max-pod-grace-period 0`
+
+|`--autoscaler-pod-priority-threshold _int_`
+|The priority that a pod must exceed to cause the cluster autoscaler to deploy additional nodes. Expects an integer, can be negative. Replace _int_ in the command with the number you want to use.
+|`integer`
+|`--autoscaler-pod-priority-threshold 0`
+
+|`--autoscaler-ignore-daemonsets-utilization`
+|Should cluster-autoscaler ignore DaemonSet pods when calculating resource utilization for scaling down.
+|`boolean`
+|Add it to set to true, do not add to set to false.
+
+|`--autoscaler-max-node-provision-time _string_`
+|Maximum time cluster-autoscaler waits for node to be provisioned. Expects string comprised of an integer and time unit (ns,us,µs,ms,s,m,h), examples: 20m, 1h.
```

Comment:
as we're already providing examples in another column:
```suggestion
|Maximum time cluster-autoscaler waits for node to be provisioned. Expects string comprised of an integer and time unit (ns,us,µs,ms,s,m,h).
```

Reply:
Fixed